### PR TITLE
Tests

### PR DESCRIPTION
**Pull request by joecohens**
// cc @GrahamCampbell

### CODE REVIEW

File: src/Arr.php [hunk 1]
```diff
@@ -23,17 +23,13 @@ class Arr
      *
      * @param array  $array
      * @param string $key
-     * @param mixed  $default
+     * @param null   $default
```
Comment (GrahamCampbell) [inline src/Arr.php:26]: Not correct.

File: src/Arr.php [hunk 2]
```diff
@@ -23,17 +23,13 @@ class Arr
...
      *
      * @return mixed
      */
-    public static function get(&$array, $key, $default)
+    public static function get(&$array, $key, $default = null)
```
Comment (GrahamCampbell) [inline src/Arr.php:30]: Could you send these changes in another pull please.

File: src/ConfigFactory.php [hunk 3]
```diff
@@ -11,10 +11,13 @@
 
 namespace StyleCI\Config;
 
+use Symfony\Component\Yaml\Yaml;
```
Comment (GrahamCampbell) [inline src/ConfigFactory.php:14]: Thanks. :)

File: src/ConfigFactory.php [hunk 4]
```diff
@@ -44,6 +47,7 @@ public function make(array $input = [])
             $config->disable($fixer);
         }
 
+        return $config;
```
Comment (GrahamCampbell) [inline src/ConfigFactory.php:50]: Please rebase. This shouldn't show up.

File: tests/ConfigFactoryTest.php [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+ */
+class ConfigFactoryTest extends AbstractTestCase
+{
+    public function testMakeConfig()
+    {
+        $config = (new ConfigFactory())->make();
+
+        $this->assertInArray('psr0', $config->getFixers());
+        $this->assertInArray('encoding', $config->getFixers());
+        $this->assertInArray('elseif', $config->getFixers());
```
Comment (GrahamCampbell) [inline tests/ConfigFactoryTest.php:31]: We should probably assert that some stuff isn't in here too.

File: src/Arr.php (hunk 1 above)
Comment (joecohens) [inline src/Arr.php:26]: You are right should remain mixed

File: src/Arr.php (hunk 2 above)
Comment (joecohens) [inline src/Arr.php:30]: If I send it on another PR tests will fail

**Comment by GrahamCampbell** [comment]
I'll manually merge in some of this. Can you send the array stuff again please.
